byoqm/visuals/dashboard.py

```diff
@@ -10,31 +12,17 @@ class Dashboard:
     def show_graphs(self):
         data = self.get_data()
         f = [line.get_line(data, key) for key in data]
-        grid = gridplot([[f[0], f[1]], [f[2], f[3]], [f[4], f[5]]])
+        grid = gridplot([[f[i], f[i + 1]] for i in range(0, len(f) - 1, 2)])
```

Comment:
Some more descriptive names here would be nice, because I have no clue what is going on.

Reply:
ok

Reply:
will changing f -> figures be enough? Else I would argue that a bit of documentation is all that is needed